spark/init.sh

```diff
@@ -130,7 +130,7 @@ else
       elif [[ "$HADOOP_MAJOR_VERSION" == "2" ]]; then
         wget http://s3.amazonaws.com/spark-related-packages/spark-$SPARK_VERSION-bin-cdh4.tgz
       else
```

Comment:
do older versions like 1.2.1 have a hadoop-2.6 artifact ? or was it not supported till recently ?